test-utils/test-utils-core/src/test/java/uk/gov/justice/services/test/utils/core/matchers/HandlerMethodMatcherTest.java

```diff
@@ -72,6 +76,11 @@ public void shouldMatchAPassThroughRequesterMethod() throws Exception {
         assertThat(TestServiceComponent.class, method("testF").thatHandles("testF").withRequesterPassThrough());
     }
 
+    @Test
+    public void shouldMatchAPassThroughRequesterMethodforPojo() throws Exception {
+        assertThat(TestServiceComponent.class, method("testK").thatHandles("testK").withRequesterPassThrough());
+    }
+
     @Test(expected = AssertionError.class)
```

Comment:
please check.. either throw exception or assert in exception block

Reply:
Already done in implementation. Method sends a command with null. Matcher.Assert will throw exception.